base/doc/ltnews40.tex

```diff
@@ -302,7 +303,24 @@ \section{Avoid code duplication in rollback}
 \githubissue{1407}
 
 
-%\section{Changes to packages in the \pkg{amsmath} category}
+
+
+\section{Changes to packages in the \pkg{amsmath} category}
+
+\section{Extend support for \cs{dots}}
+
+The implementation of \cs{dots} in \pkg{amsmath} has the feature that
+it selects different dots depending on the symbol that follows: e.g.,
+dots between commas would be normally on the baseline, while dots
```

Comment:
Sorry I missed this one earlier:
. . . would normally be . . .

is more normal, as we might say, but this is not a very important change.